testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorSmokeTest.java

```diff
@@ -236,6 +237,44 @@ public void testCreateSchema()
         assertUpdate("DROP SCHEMA " + schemaName);
     }
 
+    @Test
+    public void testRenameExistingSchema()
```

Comment:
I think these are redundant. Would something like below be equivalent?

```java
if (!hasBehaviour(SUPPORTS_RENAME_SCHEMA)) {
    // This will catch connectors who are lying about SUPPORTS_RENAME_SCHEMA - no need to create an explicit schema
    assertQueryFails(format("ALTER SCHEMA %s RENAME TO %s", session.getSchema(), session.getSchema() + randomTableSuffix()));
}

// Normal happy path test follows
String schemaName = "test_rename_schema_" + randomTableSuffix();
try {
    assertUpdate("CREATE SCHEMA " + schemaName);
    assertUpdate("ALTER SCHEMA " + schemaName + " RENAME TO " + schemaName + "_renamed");
}
finally {
    assertUpdate("DROP SCHEMA IF EXISTS " + schemaName);
    assertUpdate("DROP SCHEMA IF EXISTS " + schemaName + "_renamed");
}
```

Reply:
i.e. My expectation is that with the commit that adds the test all connectors which support rename schema but don't have SUPPORTS_RENAME_SCHEMA declared should start failing and then you can correct the declarations. No need to manually decide which connectors should have the test enabled.

Reply:
done